libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKWindowManager.m

```diff
@@ -597,6 +606,10 @@ - (void)disableWindow {
     }
 }
 
+- (void)dealloc {
+    deallocating = YES;
+}
```

Comment:
I thought you were missing [super dealloc] but you are not (https://stackoverflow.com/questions/14390652/overriding-the-dealloc-method-in-objective-c).

Reply:
thanks for double checking :)